src/pages/Search/AdvancedSearchFilters.tsx

```diff
@@ -251,6 +252,14 @@ function getFilterInDisplayTitle(filters: Partial<SearchAdvancedFiltersForm>, _:
         : undefined;
 }
 
+function shouldDisplayFilter(numberOfFilters: number, isFeatureEnabled: boolean, optionalCondition = false): boolean {
```

Comment:
the argument name `optionalCondition` is kinda confusing because of the very generic name.
In general this function feels a bit off because of it. Is it possible to not pass this argument, but instead call this function, and then in the place where you call it do something like:

```ts
const shouldDisplayXXX = shouldDisplayFilter(...) && !!singlePolicyTagLists
```
or similar?